katalog/harbor/exporter/rules.yml

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: harbor-rules
+  namespace: monitoring
+spec:
+  groups:
+  - name: harbor-rules
+    rules:
+    - alert: HarborIsDown
+      expr: harbor_up == 0
+      for: 5m
+      labels:
+        severity: critical
+      annotations:
+        summary: The service of Harbor is Down
```

please add a `decription` to the alert. The content of description will be included in the notification's body that alertmanager sends
```suggestion
        summary: The service of Harbor is Down
        description: Habor service is Down.
```